src/components/Courses/components/CourseItem/CourseItem.tsx

```diff
@@ -9,18 +9,24 @@ import { TCourses } from '../../../../types/course';
 import { useToggle } from '../../../../hooks/useToggle';
 import { getAuthorNames } from '../../../../helpers/getAuthorNames';
```

Comment:
import CourseForm from '../../../CourseForm/CourseForm'; - is not used
import Modal from '../../../../common/Modal/Modal'; - is not used